nf_core/lint/schema_lint.py

```diff
@@ -69,10 +85,16 @@ def schema_lint(self):
     # Lint the schema
     self.schema_obj = nf_core.schema.PipelineSchema()
     self.schema_obj.get_schema_path(self.wf_path)
+
     try:
-        self.schema_obj.load_lint_schema()
+        self.schema_obj.load_schema()
+        self.schema_obj.get_schema_defaults()
+        self.schema_obj.validate_schema()
```

Comment:
How come you don't use `load_lint_schema()`? Unless I'm missing something, I think that as you run the `jsonschema.validate()` function again below then there's not really any advantage to doing this and you can probably stick with just `self.schema_obj.load_lint_schema()`

Reply:
Thinking about it, it might make more sense to put all of this code into the `schema.validate_schema()` function anyway, instead of the main pipeline lint code.

Reply:
Yes you're right, I was trying it out this way because I first wasn't sure whether I first want to validate and then check defaults or vice versa - the way it is right now I can just use `load_lint_schema()` and then do the extra stuff

Thought about the too - could be a `validate_defaults()` function that belongs to the schema class. Really up to you - didn't want to mess with the schema code :)

Reply:
I think add to the schema class to keep it all together 👍🏻

Off the top of my head this code is used in some extra places, so it would mean that this stuff is checked in `nf-core schema build` as well as just `nf-core lint` which is probably a good thing.